torch/overrides.py

```diff
@@ -28,15 +30,33 @@
 import torch
 from torch._C import _is_torch_function_enabled, _disabled_torch_function_impl
 
+__all__ = [
+    "get_ignored_functions",
+    "get_testing_overrides",
```

Comment:
Related to https://github.com/pytorch/pytorch/pull/48170#issuecomment-729615574, I'd make these two private as well.

Reply:
get_ignored_functions and get_testing_overrides? I agree they should be private too.